src/main.rs

```diff
@@ -44,5 +48,8 @@ fn main() {
                 std::process::exit(1)
             }
         })
-        .start()
+        .start(match matches.values_of("interpreter") {
```

Comment:
you could do `ApplicationBuilder::start(matches.values_of("interpreter"))` and in the `start` function check perform the checks to see if its a `Option::Some` or `Option::None`